pkg/grafana/templating/templating.go

```diff
@@ -31,7 +31,10 @@ func init() {
 
 func GenerateGrafanaConfiguration(organizations []v1alpha1.GrafanaOrganization) (string, error) {
 	var orgMappings []string
-	orgMappings = append(orgMappings, fmt.Sprintf(`"*:%s:%s"`, grafana.SharedOrgName, grafanaAdminRole))
+	// TODO: We need to be admins to be able to see the private dashboards for now, remove the 2 GS groups once https://github.com/giantswarm/roadmap/issues/3696 is done.
+	orgMappings = append(orgMappings, buildOrgMapping(grafana.SharedOrg.Name, "giantswarm-ad:giantswarm-admins", grafanaAdminRole))
+	orgMappings = append(orgMappings, buildOrgMapping(grafana.SharedOrg.Name, "giantswarm-github:giantswarm:giantswarm-admins", grafanaAdminRole))
+	orgMappings = append(orgMappings, fmt.Sprintf(`"*:%s:%s"`, grafana.SharedOrg.Name, grafanaEditorRole))
```

Comment:
Please add a comment on which access rights we are configuring here. Something along the lines of

```suggestion
	// Grant Admin role to Giantswarm users logging in via ad (azure active directory).
	orgMappings = append(orgMappings, buildOrgMapping(grafana.SharedOrg.Name, "giantswarm-ad:giantswarm-admins", grafanaAdminRole))
	// Grant Admin role to Giantswarm users logging in via github.	
	orgMappings = append(orgMappings, buildOrgMapping(grafana.SharedOrg.Name, "giantswarm-github:giantswarm:giantswarm-admins", grafanaAdminRole))
	// Grant Editor role to every other users.
	orgMappings = append(orgMappings, fmt.Sprintf(`"*:%s:%s"`, grafana.SharedOrg.Name, grafanaEditorRole))
```